src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/komponenter/Filterknapp.tsx

```diff
@@ -57,7 +57,6 @@ const Filterknapp = ({ children, disabled = false, onClick, aktiv }: IFilterknap
     return (
         <StyledButton
             id={`filter_${randomUUID()}`}
-            aria-label={`filter_${randomUUID()}`}
```

Comment:
Den bør ha en aria-label, men den må være noe beskrivende for knappen :) f.eks. Historikk

Reply:
Eller hmm, nå ble jeg usikker fordi jeg ser jo skjermbilde med skjermleseren.